src/Tools/dotnet-monitor/Exceptions/ExceptionsOperation.cs

```diff
@@ -159,10 +200,13 @@ await using (Utf8JsonWriter writer = new(stream, new JsonWriterOptions() { Inden
             await stream.WriteAsync(JsonRecordDelimiter, token);
         }
 
-        private static async Task WriteText(Stream stream, IReadOnlyList<IExceptionInstance> instances, CancellationToken token)
+        private async Task WriteText(Stream stream, IReadOnlyList<IExceptionInstance> instances, CancellationToken token)
         {
-            Dictionary<ulong, IExceptionInstance> priorInstances = new(instances.Count);
-            foreach (IExceptionInstance currentInstance in instances)
+            var filteredInstances = FilterExceptions(_configuration, instances);
```

Comment:
This will cause inner exceptions to be filtered out and not reported in any manner.

Reply:
See https://github.com/dotnet/dotnet-monitor/pull/5131#discussion_r1300695223

Reply:
Latest changes add a sentinel value indicating that the inner exception was filtered out